sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/math.py

```diff
@@ -0,0 +1,19 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import math
+from functools import reduce
+from typing import List
+
+
+def list_sum(lst: List[float]) -> float:
```

Comment:
for my own understanding why are we using reduce here instead of python's built in sum?

Reply:
good call out, I've updated this